srcpkgs/sunshine/template

```diff
@@ -0,0 +1,82 @@
+# Template file for 'sunshine'
+pkgname=sunshine
+version=0.23.1
+revision=1
+archs="x86_64* aarch64*"
+build_style=cmake
+configure_args="-Wno-dev -DBUILD_TESTS=OFF -DSUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine
+ -DSUNSHINE_ASSETS_DIR=/usr/share/sunshine"
+hostmakedepends="boost cmake git make nodejs pkg-config wayland-devel"
+makedepends="boost-devel doxygen graphviz libayatana-appindicator-devel libboost_filesystem
+ libboost_locale libboost_program_options libcap-devel libcurl-devel libdrm-devel libevdev-devel
+ libnuma-devel libnotify-devel libopusenc-devel libva-devel libvdpau-devel libX11-devel
+ libxcb-devel libXcursor-devel libXfixes-devel libXi-devel libXinerama-devel libXrandr-devel
+ libXtst-devel MesaLib-devel miniupnpc-devel nv-codec-headers opus-devel openssl-devel
+ pulseaudio-devel wayland-protocols"
+short_desc="Self-hosted GameStream host for Moonlight"
+maintainer="myothk <myoothkyaw@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://app.lizardbyte.dev/Sunshine"
+distfiles="https://github.com/LizardByte/Sunshine/archive/refs/tags/v${version}.tar.gz
+ https://github.com/michaeltyson/TPCircularBuffer/archive/refs/tags/1.6.2.tar.gz>TPCircularBuffer.tar.gz
+ https://github.com/LizardByte/Virtual-Gamepad-Emulation-Client/archive/8d71f6740ffff4671cdadbca255ce528e3cd3fef.tar.gz>ViGEmClient.tar.gz
+ https://github.com/LizardByte/build-deps/archive/efd3a380113e8ae98ae68cc1d73fd7c4b54b03c6.tar.gz>build-deps.tar.gz
+ https://github.com/google/googletest/archive/refs/tags/v1.14.0.tar.gz>googletest.tar.gz
+ https://github.com/moonlight-stream/moonlight-common-c/archive/cbd0ec1b25edfb8ee8645fffa49ff95b6e04c70e.tar.gz>moonlight-common-c.tar.gz
+ https://github.com/sleepybishop/nanors/archive/e9e242e98e27037830490b2a752895ca68f75f8b.tar.gz>nanors.tar.gz
+ https://github.com/FFmpeg/nv-codec-headers/archive/22441b505d9d9afc1e3002290820909846c24bdc.tar.gz>nv-codec-headers.tar.gz
+ https://github.com/LizardByte/nvapi-open-source-sdk/archive/refs/tags/R535.tar.gz>nvapi-open-source-sdk.tar.gz
+ https://github.com/LizardByte/tray/archive/4d8b798cafdd11285af9409c16b5f792968e0045.tar.gz>tray.tar.gz
+ https://gitlab.freedesktop.org/wayland/wayland-protocols/-/archive/1.35/wayland-protocols-1.35.tar.gz>wayland-protocols.tar.gz
+ https://gitlab.freedesktop.org/wlroots/wlr-protocols/-/archive/2b8d43325b7012cc3f9b55c08d26e50e42beac7d/wlr-protocols-2b8d43325b7012cc3f9b55c08d26e50e42beac7d.tar.gz>wlr-protocols.tar.gz
+ https://gitlab.com/eidheim/Simple-Web-Server/-/archive/27b41f5ee154cca0fce4fe2955dd886d04e3a4ed/Simple-Web-Server-27b41f5ee154cca0fce4fe2955dd886d04e3a4ed.tar.gz>Simple-Web-Server.tar.gz
+ https://github.com/cgutman/enet/archive/04e27590670a87a7cd40f5a05cda97467e4e25a3.zip>enet.tar.gz"
```

Comment:
don't hardcode-the submodule versions directly in the distfile urls. something like this would be better, and also means it won't overwrite old versions of these distfiles when updated:
```bash
# after revision
_tpc_ver=1.6.2
_vgec_ver=8d71f6740ffff4671cdadbca255ce528e3cd3fef
# ...
# then
distfiles="https://github.com/LizardByte/Sunshine/archive/refs/tags/v${version}.tar.gz
 https://github.com/michaeltyson/TPCircularBuffer/archive/refs/tags/${_tpc_ver}.tar.gz>TPCircularBuffer-${_tpc_ver}.tar.gz
 https://github.com/LizardByte/Virtual-Gamepad-Emulation-Client/archive/${_vgec_ver}.tar.gz>ViGEmClient-${_vgec_ver}.tar.gz
 ..."
# and
skip_extraction="
 TPCircularBuffer-${_tpc_ver}.tar.gz
 ..."
```